architecture-docs/adr-003.md

```diff
@@ -164,6 +166,7 @@ Accepted
 (or equivalent deterministic-finality consensus algorithms), but for probabilistic-finality consensus algorithms
 - One more repository
 - One "rocksdb" instance (as the nodes will be stored in one column rather than in a separate database instance)
+- The block height to root hash mapping is changed to block height to version number for ABCI querying
```

Comment:
this one may be neutral, as other storage may need this, or I guess this is compared with the alternative of changing jellyfish implementation?

Reply:
I changed it into:
```
When query history staking state, convert the request block height to version number through history ChainNodeState first
```
It is in neutral.